analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DeadStores.CSharp11.cs

```diff
@@ -114,9 +114,20 @@ void NewlinesInStringInterpolation(string param)
 
     string empty = "";
     string x = $"{empty +
-        empty}"; // Noncompliant@-1 FP (string is still empty, should be compliant)
+        empty}"; // Noncompliant@-1 string interpolation values are intentionally not evaluated
     x = "Test";
     Foo(x);
 }
 
+void IgnoredValues()
+{
+    string emptyMultilineRawStringLiteral = $$"""
```

Comment:
this is not empty either, it contains many spaces and newlines, please rename

![image](https://user-images.githubusercontent.com/38876598/201037601-a06c9d89-1841-4388-9e7d-42afa157cf9f.png)

Reply:
Please check: https://github.com/SonarSource/sonar-dotnet/issues/6301#issuecomment-1311870738 and https://github.com/SonarSource/sonar-dotnet/issues/6301#issuecomment-1311871759